server-spi-private/src/main/java/org/keycloak/cookie/CookieScope.java

```diff
@@ -1,17 +1,17 @@
 package org.keycloak.cookie;
 
-import org.keycloak.common.util.ServerCookie;
+import jakarta.ws.rs.core.NewCookie;
 
 public enum CookieScope {
     // Internal cookies are only available for direct requests to Keycloak
-    INTERNAL(ServerCookie.SameSiteAttributeValue.STRICT, true),
+    INTERNAL(NewCookie.SameSite.STRICT, true),
 
     // Federation cookies are available after redirect from applications, and are also available in an iframe context
     // unless the browser blocks third-party cookies
-    FEDERATION(ServerCookie.SameSiteAttributeValue.NONE, true),
+    FEDERATION(NewCookie.SameSite.NONE, true),
 
     // Federation cookies that are also available from JavaScript
-    FEDERATION_JS(ServerCookie.SameSiteAttributeValue.NONE, false),
+    FEDERATION_JS(NewCookie.SameSite.NONE, false),
```

Comment:
Perhaps `CLIENT`?

Reply:
I thought about that, but it's not really cookies that are available to clients, rather cookies that are available under federation oauth/saml requests.

Reply:
I see. I thought these were related to things like session management so they are indeed cookies that we want to make accessible to clients (httpOnly=false) and cross-origin.